docs/usage/index.md

```diff
@@ -35,8 +35,10 @@ application.
 
 ```{note}
 To avoid confusion, note that:
- - `cdn.jsdelivr.net/pyodide/` distributes Python packages built with Pyodide as well as `pyodide.js`
- - `cdn.jsdelivr.net/npm/pyodide@0.19.0/` is a mirror of the Pyodide NPM package, which includes none of the WASM files
+ - `cdn.jsdelivr.net/pyodide/` distributes Python packages built with Pyodide as well
+    as `pyodide.js`
+ - `cdn.jsdelivr.net/npm/pyodide@0.19.0/` is a mirror of the Pyodide NPM package which
```

Comment:
Maybe we should make the version here update with the current Pyodide version too.

Reply:
> which includes none of the WASM files

Should have always been "includes none of the wheel files".

Reply:
> Maybe we should make the version here update with the current Pyodide version too.

sounds good to me.